### DISCUSSION TITLE
Where can I download the old version for Firefox 56 and lower?

### DISCUSSION THREAD

**Discussion by John-foobar-Doe** [foxyproxy/firefox-extension#175] [Q&A] [answered]
I have a Firefox profile used with Firefox ESR 52.8.1, and it accidentally upgraded itself to 60, which triggered and upgrade of foxyproxy to version 7.5.1. But now when reverting back to Firefox ESR 52.8.1, foxyproxy 7.5.1 is not compatible and disabled. But I couldn't find anywhere to download foxyproxy older version's .xpi. Could you please provide a link? Thanks a lot!

**Comment by ericjung** [comment, marked as answer]
hello,

You can get it here:

https://github.com/foxyproxy/firefox-extension/tree/master/archive/dist/foxyproxy-standard/4.6.5

**Reply by John-foobar-Doe** [reply]
Thanks a lot!
